dev_tools/scripts/config

```diff
@@ -6,10 +6,26 @@
     - FixInclude: "hip/hip_runtime.h"
```

Comment:
What's the purpose of this file?

Reply:
You mean hip_runtime? It contains all the macros like hipLaunchKernelGGL, HIP_KERNEL_NAME, device-side assert etc.

Reply:
No, I meant this entire `config` file, haha. I guess it's unrelated to this PR but I have no idea what's going on here.

Reply:
This file configures the `format_header.sh` script by fixing a few special-cases where the default detection of the main header belonging to a file (the one that is included at the top) doesn't work properly